### ISSUE TITLE
Component Request - gallery-text - 3hrs

### ISSUE THREAD

**Issue by drewbaker** [funkhaus/factory#8]
## Component Description

Some text with logos in it. Hover on each logo brings up a cross fading slideshow of rapid images.

Note the "EXPLORE ->" hover state too.

## Design

Please also see attached screenshots for quick reference.

- Desktop: https://xd.adobe.com/view/1b5027a3-98f1-48ce-7968-22ffdcb8d2fe-6d61/screen/84141268-0730-442d-b88f-96480f3fadd2/Make-Make-Landing-4

If no mobile designs provided, please use your best judgment for responsiveness.

Email drew@funkhaus.us for the design password.

## Props

I'd expect the text and logos to be hardcoded into this component. By it's nature, it's hard to imagine how to place the logos and have them have the slideshow hover states.

```
props: {
    rpsImages: {
        // Mock: api.images. An array of images used in each slideshow.
        type: Array,
        default: () => []
    },
    // And repeated for each company logo
    elasticImages = []
    primaryImages = []
    a52Images = []
    jaxImages = []
    indestructibleImages = [] 
}
```

## Required components

1. SVGs for each company.
2. Maybe need a component for the crossfading slideshow? Maybe not.

## Time & Budget

Time estimate: 4hrs
Budget: $400

![Screen Shot 2020-06-26 at 5 52 17 PM](https://user-images.githubusercontent.com/475015/85910982-e3be3800-b7d6-11ea-83e9-253b0aaac1ab.png)
![Screen Shot 2020-06-26 at 5 52 43 PM](https://user-images.githubusercontent.com/475015/85910983-e456ce80-b7d6-11ea-8ace-20be73c603ef.png)
![Screen Shot 2020-06-26 at 5 52 51 PM](https://user-images.githubusercontent.com/475015/85910985-e4ef6500-b7d6-11ea-8c64-7dbe15792d17.png)

**Comment by ibanzajoe** [comment]
@drewbaker Hey Drew.  I took a stab at this one a while back.  Don't know if you saw it or not.

**Comment by drewbaker** [comment]
Yes sorry, was slammed at work. Will review today and also add some more component request in.
